src/stores/EndpointConfig/Store.ts

```diff
@@ -183,6 +175,10 @@ const getInitialState = (
     },
 
     setServerUpdateRequired: (updateRequired) => {
+        if (get().serverUpdateRequired === updateRequired) {
+            return;
+        }
```

Comment:
Why was this added?

Reply:
This was addressed in Slack on Apr. 23rd. You stated that noise for setters of scalar state properties is acceptable, depending on the side effects. Consequently, this change has been reversed.